server/src/utils/error-handler.ts

```
@@ -0,0 +1,48 @@
+ * Standardizes how we log errors so that the error is formated in a useful way
+ * in datadog.
+ * @param logger
+ * @param error
+ * @param message - optional message
+ * @param extraMetadata - additional metadata to log
+ */
+export const logError = (
+  logger: Logger,
+  error: unknown,
+  message?: string,
+  extraMetadata?: {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)